tests/Integration/UI/NetworkAdminSitesListTest.php

```diff
@@ -80,8 +80,10 @@ public function test_api_key_column_is_present(): void {
 	 * @return void
 	 */
 	public function test_api_key_column_is_correctly_printed(): void {
-		$blog_id_with_api_key    = $this->factory->blog->create();
-		$blog_id_without_api_key = $this->factory->blog->create();
+		$blog_id_with_api_key = $this->factory->blog->create();
+
```

Comment:
I would remove this empty line (L84) so the related lines are together.

Reply:
I don't necessarily agree. Since the second statement has a comment above, I aesthetically prefer to leave one line break before it.